build/register.js

```diff
@@ -14,13 +14,13 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
  */
-var Promise, postAsync, randomstring;
+var Promise, fetch, randomstring;
```

Comment:
Could you chuck in the gulp header thing to silence this file, see https://github.com/resin-io/resin-sdk/pull/236/files#r91601139